### PR TITLE
[#GB4] Added profile avatar uploader

### PR DESCRIPTION
**Pull request by kris-kolosek**

### CODE REVIEW

File: app/uploaders/avatar_uploader.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  end
+
+  # Add a white list of extensions which are allowed to be uploaded.
+  # For images you might use something like this:
+  # def extension_whitelist
+  #   %w(jpg jpeg gif png)
+  # end
+
+  # Override the filename of the uploaded files:
+  # Avoid using model.id or version_name here, see uploader/store.rb for details.
```
Comment (sniffler) [inline app/uploaders/avatar_uploader.rb]: Line is too long. [81/80]

File: app/uploaders/avatar_uploader.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn"t been a file uploaded:
+  #def default_url
+  #  ActionController::Base.helpers.asset_path("images/" + [version_name, "default.png"].compact.join("_"))
+  #end
```
Comment (sniffler) [inline app/uploaders/avatar_uploader.rb]: Missing space after #.

File: app/uploaders/avatar_uploader.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn"t been a file uploaded:
+  #def default_url
+  #  ActionController::Base.helpers.asset_path("images/" + [version_name, "default.png"].compact.join("_"))
```
Comment (sniffler) [inline app/uploaders/avatar_uploader.rb]: Line is too long. [107/80]

File: app/uploaders/avatar_uploader.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn"t been a file uploaded:
+  #def default_url
```
Comment (sniffler) [inline app/uploaders/avatar_uploader.rb]: Missing space after #.

File: app/helpers/profiles_helper.rb [hunk 5]
```diff
@@ -2,4 +2,11 @@ module ProfilesHelper
   def current_profile
     @_profile ||= current_user.profile
   end
+  def edit_image(avatar_url)
```
Comment (sniffler) [inline app/helpers/profiles_helper.rb]: Use empty lines between method definitions.

File: app/assets/stylesheets/components/devise/_login.scss [hunk 6]
```diff
@@ -30,7 +30,7 @@
     width: 100%;
   }
 
-  >.signbox>.forgotpass>.link {
+  >.signbox>.forgotpass> {
```
Comment (sniffler) [inline app/assets/stylesheets/components/devise/_login.scss]: Selector should have depth of applicability no greater than 2, but was 3

**Comment by kolosek** [comment]
@kris-kolosek for current_profile method, see if Helper is the best place to write it. See where is current_user is defined. Even if you need to go into gem.

**Pull request merged**